tests/console/lvm_thin_check.pm

```diff
@@ -1,4 +1,4 @@
-# SUSE's openQA tests
+# Suse'S openQA tests
```

Comment:
Not very important but this is new? I was not aware of such name change, as SUSE is an acronym.

Reply:
this is accidentally actually. i reverted